packages/veritone-redux-common/.eslintrc.js

```diff
@@ -1,22 +1,24 @@
 const env = require('veritone-dev-env');
 
-module.exports = {
-  ...env.eslintReact,
-  globals: {
-    ...env.eslintReact.globals,
-    module: true,
-    process: true
-  },
-  env: {
-    ...env.eslintReact.env,
-    jest: true
-  },
-  plugins: [
-    ...env.eslintReact.plugins,
-    'import'
-  ],
-  rules: {
-    ...env.eslintReact.rules,
-    'import/order': 2
-  }
-};
+module.exports = Object.assign({}, env.eslintReact, {
+  globals: Object.assign(
+    {
```

Comment:
these need to be reversed, so `({}, globals, { local overrides })` to match the current behavior

idea is to be able to override globals in local configs

Reply:
updated